api/src/test/java/org/openmrs/module/sync/api/db/hibernate/HibernateSyncDAOTest.java

```diff
@@ -13,23 +13,18 @@
  */
 package org.openmrs.module.sync.api.db.hibernate;
 
-import java.lang.reflect.Method;
-import java.util.Properties;
-
 import org.junit.Assert;
-import org.junit.Test;
-import org.junit.runner.RunWith;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
 import org.mockito.Mockito;
 import org.openmrs.api.context.Context;
 import org.powermock.api.mockito.PowerMockito;
-import org.powermock.core.classloader.annotations.PrepareForTest;
-import org.powermock.modules.junit4.PowerMockRunner;
+
+import java.util.Properties;
 
 /**
  * Tests methods in HibernateSyncDAO
  */
-@RunWith(PowerMockRunner.class)
```

Comment:
My change I commented on earlier allowed me to get rid of all of the mocking, which was failing.